Formula/portable-ruby@2.2.rb

```diff
@@ -64,8 +89,8 @@ def install
     args << "--with-opt-dir=#{paths.join(":")}"
 
     system "./configure", *args
-    system "make"
-    system "make", "install"
+    make
+    make "install"
```

Comment:
If we run into another non-ruby formula in this tap that needs it, I'll convert everything over!